lntest/bitcoind.go

```diff
@@ -0,0 +1,177 @@
+// +build bitcoind
+
+package lntest
+
+import (
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/btcsuite/btcd/rpcclient"
+)
+
+// logDir is the name of the temporary log directory.
+const logDir = "./.backendlogs"
```

Comment:
IIRC, this is defined elsewhere already?

Reply:
It is re-defined since it is behind a different build flag.